### ISSUE TITLE
Setting REMOVE_VOLUMES=1 causes docker-gc to exit with status 1

### ISSUE THREAD

**Issue by makkes** [spotify/docker-gc#201]
## Description

When setting `REMOVE_VOLUMES=1` and there are no volumes found in Docker then `docker-gc` exits with status code 1 because of `grep` returning 1 when it has not found any matches.

## How to reproduce

1. Make sure that no Docker volumes exist on the system
1. Run `REMOVE_VOLUMES=1 docker-gc`

## What do you expect

`docker-gc` should exit with status code 0.

## What happened instead

`docker-gc` exits with status code 1.

## Software:

- bash version: Any

## Full backtrace

```text
$ docker volume ls
DRIVER              VOLUME NAME
$ sudo REMOVE_VOLUMES=1 ./docker-gc
$ echo $?
1
```

**Comment by makkes** [comment]
The main problem we have with this currently is that we call `docker-gc` from a systemd unit and systemd then marks the unit as failed.

**Comment by makkes** [comment]
@mattnworb @davidxia There's an open PR to fix this issue (#202) that is waiting to be merged. Would you be able to merge the PR or provide feedback, please?

**Comment by davidxia** [comment]
Sorry for the delay. I just merged it.
